dist/ParameterValidator.js

```diff
@@ -102,15 +102,17 @@ var ParameterValidator = function () {
        *								- If an item is an Object, it's assumed that the object's only key is the name of a parameter to be validated
        *								 and its corresponding value is a function that returns true if that parameter's value in paramsProvided is
        *								 valid.
-       * @param    {Object}    [extractedParams] - This method returns an object containing the names and values of the validated parameters extracted.
```

Comment:
You can ignore the changes to these files in the /dist directory, since they're just the transpiled versions of the changes in the /src files.